packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.stories.tsx

```diff
@@ -10,15 +10,15 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { Story } from "@storybook/react";
-import { Chip, Stack } from "@okta/odyssey-react-mui";
+import { Meta, Story } from "@storybook/react";
+import { Tag, TagList, TagProps } from "@okta/odyssey-react-mui";
 import { MuiThemeDecorator } from "../../../../.storybook/components";
 
 import TagMdx from "./Tag.mdx";
 
 export default {
   title: `MUI Components/Tag`,
-  component: Chip,
+  component: Tag,
   parameters: {
     docs: {
       page: TagMdx,
```

Comment:
Let's do that thing where we make this:
```
const storybookMeta: Meta<TagProps> = {};
export default storybookMeta;
```

Reply:
Happy to change it, but what's the rationale of doing it this way vs the current:

```
export default {
  ...
} as Meta<TagProps>;
```